app/javascript/components/App.js

```diff
@@ -1,11 +1,26 @@
 import React, { Component } from 'react'
+import Home from './pages/Home'
 import Header from './components/Header'
+import Navigation from './components/Navigation'
+import Apartments from './pages/Apartments'
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom'
 
 class App extends Component {
   render() {
     return (
       <>
         <Header {...this.props} />
+        <Router>
+          <Navigation />
+          <Switch>
+            <Route exact path="/" component={Home} />
+            <Route path="/Apartments" component={Apartments} />
+          </Switch>
+        </Router>
```

Comment:
Router should be the top level tag and header should be nested inside of it. Router > header> Switch